spec/select_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'select.rb'
+
+describe Select do
+		let(:burger){double (:burger)}
+		it 'shows the menu' do 
+			men = {'Burger'=> 10, 'Pint'=> 4.50,'Steak'=>18,'Wine'=> 22,'Chips'=> 3.25}
+			expect(subject.menu).to eq men
+		end
+		it 'shows the menu' do 
+			men = {'Burger'=> 10, 'Pint'=> 4.50,'Steak'=>18,'Wine'=> 22,'Chips'=> 3.25}
+			expect(subject.see).to eq men
+		end
+
+		it 'has an order' do
+ 			expect(subject).to respond_to(:order)
+		end
+		it 'has an due amount' do
+ 			expect(subject).to respond_to(:order)
+ 		end
```

Comment:
Lint/BlockAlignment: `end` at 19, 3 is not aligned with `it 'has an due amount' do` at 17, 2.